src/components/Registration/AgencyRegistration.js

```diff
@@ -0,0 +1,837 @@
+    backgroundColor: '#2D5496',
+    '&:hover': {
+      backgroundColor: '#2D5496',
+    },
+    width: '50%',
+  },
+  stepper: {
+    margin: 'auto',
+    width: '60%',
+  },
+  smallButton: {
+    color: 'white',
+    background: '#2D5496',
+    borderRadius: '6px',
+  },
+  formControl: {
+    margin: 1,
+    width: 550,
+  },
+  registrationFruit2: {
+    position: 'absolute',
+    right: '-10px',
+    top: '160px',
+    transform: 'matrix(0.7, 0.47, -0.42, 0.7, 0, 0)',
+  },
+  registrationFruit3: {
+    position: 'absolute',
+    right: '0px',
+    top: '60px',
+    transform: 'scale(0.6)',
+  },
+  confirmationFruit2: {
+    position: 'absolute',
+    right: '150px',
+    top: '100px',
+    transform: 'matrix(0.7, 0.47, -0.42, 0.7, 0, 0)',
+  },
+  confirmationFruit3: {
+    position: 'absolute',
+    right: '170px',
+    top: '0px',
+    transform: 'scale(0.6)',
+  },
+  picContainer: {
+    position: 'relative',
+  },
+  emptyGrid: {
+    height: '200px',
+    position: 'relative',
+  },
+});
+
+const INITIAL_FORM_STATE = {
+  contactName: '',
+  agencyName: '',
+  taxID: '',
+  website: '',
+  email: '',
+  phone: '',
+  addOne: '',
+  addTwo: '',
+  additionalContact: '',
+  socials: '',
+  additionalComments: '',
+  password: '',
+  confirmPassword: '',
+  org: '',
+  zipcode: '',
+  comments: '',
+  popServed: [],
+  userID: '',
+};
+
+function getSteps() {
+  return ['Step 1', 'Step 2', 'Confirmation'];
+}
+
+export default function RegistrationScreen() {
+  const reactTheme = useTheme();
+  const [formState, setFormState] = useState(INITIAL_FORM_STATE);
+  const [loading, setLoading] = useState(false);
+  const [errorMsg, setErrorMsg] = useState('');
+  const [currentStep, setCurrentStep] = useState(1);
+
+  const classes = useStyles();
+
+  useEffect(() => {
+    base('Users').find(store.getState().userData.user.fields['user id']).then((record) => {
+      setFormState(
+        {
+          ...formState,
+          agencyName: record.fields.organization,
+          zipcode: record.fields.zipcode,
+          contactName: record.fields['contact name'],
+          phone: record.fields.phone,
+          email: record.fields.username,
+          userID: record.fields['user id'],
+        },
+      );
+    });
+  }, []);
+
+  const handleChange = useCallback((event) => {
+    event.preventDefault();
+    setFormState(
+      {
+        ...formState,
+        [event.currentTarget.name]: event.currentTarget.value,
+      },
+    );
+  }, [formState, setFormState]);
+
+  const handleSelect = useCallback((event) => {
+    event.preventDefault();
+    setFormState(
+      {
+        ...formState,
+        [event.target.name]: event.target.value,
+      },
+    );
+  }, [formState, setFormState]);
+
+  const handleSubmit = useCallback((event) => {
+    event.preventDefault();
+    setLoading(true);
+    try {
+      base('Agencies').create([
+        {
+          fields: {
+            'farm name': formState.organization,
```

Comment:
Oh yeah oops I'll fix that. Also the 'farm name' column in the airtable is the id for the farm table. I'll add a 'legal name' column but I don't want to delete the 'farm name' column since it might break someone else's feature before demo day lol. An issue with this is that 'farm name' is the farm table's "primary field" where it acts as a unique identifier. So I think I'll have to add a redundant 'farm name': formState.farmName in my base.create call in ProdRegistration.js. Just wanted to lyk